common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceMinificationServiceImpl.java

```diff
@@ -107,44 +100,46 @@ public Resource minify(Resource originalResource) {
     @Override
     public Resource minify(Resource originalResource, String filename) {
         if (!getEnabled()) {
-            LOG.trace("Minification is disabled, returning original resource");
+            LOG.trace(MINIFICATION_IS_DISABLED_RETURNING_ORIGINAL_RESOURCE);
             return originalResource;
         }
-        
+
         String type = getFileType(originalResource, filename);
         if (type == null) {
-            LOG.info("Unsupported minification resource: " + filename);
+            LOG.trace("Unsupported minification resource: " + filename);
```

Comment:
Why not continue to `LOG.info` here?

Reply:
Info is fine. I didn't realize that was the original logging level. Since ReferenceSite has info logging on it seemed extra loud since this method gets called for image resources as well as JS, and CSS